SPECS/sm.spec

```diff
@@ -1,17 +1,17 @@
-%global package_speccommit ffb71ac33560eadc29827c13157837f60bc74beb
-%global package_srccommit v3.2.0
+%global package_speccommit 9b3d31b2bff0ffe8b70f0a49a857aaf910977f1c
+%global package_srccommit v3.2.1
 
 # -*- rpm-spec -*-
 
 Summary: sm - XCP storage managers
 Name:    sm
-Version: 3.2.0
-Release: 1.5%{?xsrel}%{?dist}
+Version: 3.2.1
+Release: 1.1%{?xsrel}%{?dist}
 License: LGPL
 URL:  https://github.com/xapi-project/sm
-Source0: sm-3.2.0.tar.gz
+Source0: sm-3.2.1.tar.gz
 
-%define __python python3.6
+%define __python python3
```

Comment:
This came for the merge of `XS-8.3` IDK if we want to keep this modification or not... @stormi ?

Reply:
it does :+1: